Source/WebCore/platform/graphics/adwaita/ControlAdwaita.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2024 Sony Interactive Entertainment Inc.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY APPLE INC. ``AS IS'' AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL APPLE INC. OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+ * OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#pragma once
+
+#if USE(THEME_ADWAITA)
+
+#include "Adwaita.h"
+#include "ControlPart.h"
+#include "PlatformControl.h"
+
+namespace WebCore {
+
+class ControlFactoryAdwaita;
+class Path;
+
+class ControlAdwaita : public PlatformControl {
+public:
+    ControlAdwaita(ControlPart&, ControlFactoryAdwaita&);
+
+protected:
+    static Color getAccentColor(const ControlStyle&);
```

Comment:
```suggestion
    static Color accentColor(const ControlStyle&);
```

Reply:
Why is this one here but systemAccentColor and others are global in an Adwaita namespace?

Reply:
`Adwaita.h` is included from some places.
Only this function depends on `ControlStyle` class. I just reduced the number of headers to be included recursively.
But, not important, nor a strong opinion. Which do you prefer, ControlAdwaita class static functions or functions in Adwaita namespace?
I will rename getAccentColor.

Reply:
It's ok, if it reduces the header includes.